coiled_calib.py

```diff
@@ -24,27 +23,25 @@ def calibrate(self, calib_pars=None, load=False, tidyup=True, **kwargs):
             verbose (bool): whether to print output from each trial
             kwargs (dict): if supplied, overwrite stored run_args (n_trials, n_workers, etc.)
         """
+        import optuna as op
+
         # Load and validate calibration parameters
         if calib_pars is not None:
             self.calib_pars = calib_pars
         self.run_args.update(kwargs) # Update optuna settings
 
-        # Run the optimization
-        t0 = sc.tic()
-        self.study = self.make_study()
-
-        #self.run_workers() # <-- This is the line from the base class to run locally using multiprocessing
-        ################################
         cluster = coiled.Cluster(
             n_workers=10,
-            name='StarsimCalibrationOnCoiled'
+            name='StarsimCalibrationOnCoiled',
         )
+        client = cluster.get_client()
 
-        # backend_storage = op.storages.InMemoryStorage() --> not sure this is needed
-        dask_storage = op.integration.DaskStorage(storage=None,
-                                                  client=cluster.get_client())
+        # Run the optimization
+        t0 = sc.tic()
+        self.run_args.storage = op.integration.DaskStorage(storage=None, client=client)
```

Comment:
I assigned the `DaskStorage` object to `self.run_args.storage` to the `self.make_study()` call below would pick up that we want to use the non-default `DaskStorage` here. There may be a better way to do that -- I was just trying to get something that works.